extension/scripts/features/cleanflight/ttCleanFlight.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+(async () => {
```

Comment:
Should probably check that you are actually flying? Not yet tested if this is an issue though.

Reply:
Shouldn't trigger while abroad either.

Reply:
You are still registering the feature, instead of preventing it from even registering.